controllers/constants/images.go

```diff
@@ -12,7 +12,7 @@ var (
 
 	RekorRedisImage    = "registry.redhat.io/rhtas/trillian-redis-rhel9@sha256:8c12d98f96ed3daa3ea7ce03d52e8b6748c4b7ab20eee98c1d1ad031d14af00e"
 	RekorServerImage   = "registry.redhat.io/rhtas/rekor-server-rhel9@sha256:133e42c7dd44ce6d12a3cee28ead1a4eb1a99ebaaa07d67d4362eec7561ae672"
-	RekorSearchUiImage = "registry.redhat.io/rhtas/rekor-search-ui-rhel9@sha256:0c8a24911435b3d8b9c137e8105192a3f513d775cc2728a87506c68785975f7b"
+	RekorSearchUiImage = "registry.redhat.io/rhtas/rekor-search-ui-rhel9sha256:047b9018425c1bacd0919b88b86af2f053bd14d6aafb63f6105221a9045d517c"
```

Comment:
```suggestion
	RekorSearchUiImage = "registry.redhat.io/rhtas/rekor-search-ui-rhel9@sha256:047b9018425c1bacd0919b88b86af2f053bd14d6aafb63f6105221a9045d517c"
```

Just missing the `@` here, ill Create a new pr and close this